saleor/graphql/product/types/products.py

```diff
@@ -374,6 +374,13 @@ def __resolve_reference(root, _info, **_kwargs):
         return graphene.Node.get_node_from_global_id(_info, root.id)
 
 
+class StockInput(graphene.InputObjectType):
```

Comment:
Most of input types in Saleor are defined in mutation files since we use them only for mutations. Maybe we could move this to `graphql/product/mutations/products.py`?

Reply:
Ok, I'll move that.